tst/com/amazon/kinesis/streaming/agent/processing/processors/DataConverterTest.java

```diff
@@ -195,12 +195,15 @@ public void testSysLogDataConverter() throws Exception {
         final String expectedLogWithoutPID = "{\"timestamp\":\"Mar 12 12:01:02\",\"hostname\":\"server4\",\"program\":\"snort\",\"processid\":null,\"message\":\"Ports to decode telnet on: 21 23 25 119\"}\n";
         verifyDataConversion(converter, logWithoutPID.getBytes(), expectedLogWithoutPID.getBytes()); 
     }
-    
-    private void verifyDataConversion(IDataConverter converter, byte[] dataBin, byte[] expectedBin) throws Exception {
-        ByteBuffer data = ByteBuffer.wrap(dataBin);
-        ByteBuffer res = converter.convert(data);
-        byte[] resBin = new byte[res.remaining()];
-        res.get(resBin);
-        Assert.assertEquals(resBin, expectedBin);
```

Comment:
Thanks @wslulciuc for the offer of help!! That'd be super nice. We have our own pipeline of running these unit tests before releasing the source code to Github. We use ant target to resolve the dependencies, compile the code, and kick off the test cases. Our internal tool has some integration and pre-defined targets that help us run these test cases pretty easily. I haven't tried running them out of the box so I imagined that could involve some efforts. How were you getting it running? Would you able to provide the script or target that you were using? Or any support you'd like me to provide?